nix/overlays.nix

```diff
@@ -28,10 +28,12 @@ in {
     (final: prev: let
       date = mkDate (self.lastModifiedDate or "19700101");
     in {
-      hyprland = prev.callPackage ./default.nix {
+      hyprland = final.callPackage ./default.nix {
         stdenv = prev.gcc13Stdenv;
         version = "${props.version}+date=${date}_${self.shortRev or "dirty"}";
+        wlroots = prev.wlroots-hyprland;
```

Comment:
This is till using `prev`

Reply:
I don't know what to do to this, it shows all updated on my end. Does this have to do with my force-push?

Reply:
It now says out-dated. It didn't at the time I went to the "Files changed" tab and started a review. Weird.